cloud/shared/bin/lib/terraform.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import os
+import shutil
+
+from cloud.shared.bin.lib import civiform_mode
+
+
+def perform_apply():
+    '''Generates terraform variable files and runs terraform init and apply.'''
+
+    TERRAFORM_PLAN_OUT_FILE = "terraform_plan"
+    TERRAFORM_TEMPLATE_DIR = os.getenv("TERRAFORM_TEMPLATE_DIR")
```

Comment:
For these three variables, there's no need to use UPPER_CASE. Instead we can use the Python default lower_underscore_case.